.github/workflows/push.yml

```diff
@@ -56,3 +56,27 @@ jobs:
 
       - name: Fail on differences
         run: git diff --exit-code
+
+  docs:
+    name: Build Docusaurus
+    runs-on:
+      group: databrickslabs-protected-runner-group
+      labels: linux-ubuntu-latest
```

Comment:
please run in on simple ubuntu-latest, no need to run it on protected ones since we're not deploying anything.

Reply:
updated